app/(main)/login/page.tsx

```diff
@@ -36,17 +47,19 @@ type LoginUserSchemaType = z.infer<typeof LoginUserSchema>;
 
 /**
  * Renders the login page.
- * @returns {JSX.Element} The rendered login page.
+ * @returns {React.JSX.Element} The rendered login page.
  */
-const Login = (): JSX.Element => {
+const Login = (): React.JSX.Element => {
```

Comment:
Import `JSX` and you won't need `React.` infront of it.

Reply:
Done